### ISSUE TITLE
Match Report: Expand Injury Dropdown to allow for Decay

### ISSUE THREAD

**Issue by mfranchetti** [TheNAF/naflm#142]
Players with Decay can be affected by two Injury results from a single match. At the moment, the Match Report (and OBBLM) does not support the recording and handling of this.
Currently, this is only an issue for two players positions (Khemri: Tomb Guardians; Nurgle: Rotters) and only when both injuries are SIs, so only 1/36 casualties. In any other case the SI will be overruled by Death or make the BH/MNG irrelevant.

Proposed solution is to alter the Injury dropdown for those specific players to include the possible combinations:
image
It increases the options from 8 to 23 but I suspect it might be easier to manage on the back end and would affect a minority of players.

**Comment by Shteve0** [comment]
I think better to do it where the player has decay, rather than for those
specific players? May be edge case but as it's a result of the skill and
not tied to the positional it would be better supported in the event of
roster changes or additions down the line (or where skills are abnormally
allocated by a commissioner)

**Comment by mfranchetti** [comment]
Yes, agreed and that was my original intention. I was just flagging the players to highlight that it is a low impact issue.

**Comment by Shteve0** [comment]
Gotcha :)
